README.md

```diff
@@ -65,6 +65,39 @@ will assist in catching regressions if the `update` function begins to set other
 values or if the view code does not capture newly-added fields to the `User`
 schema.
 
+For the sake of example, if the controller action under test returned a `User`
+with the same keys and values differing only with the value for id, then the
+test would fail and the output may look like the example below:
+
+```
+  1) test success: it returns a 200 and a newly updated User (UserControllerTest)
+     test/user_controller_test.exs:294
+     There were issues the comparison:
```

Comment:
Suggestion, blocking (but not for the reasons you would expect)

```suggestion
     There were issues with the comparison:
```
Through reviewing, I notice that there is a typo with the test output here. We're missing the word `with`.
[See the offending code here](https://github.com/Shimmur/checker_cab/blob/4a62ccbd07d103cc9967428f53b260e0bd2a2f26/lib/checker_cab.ex#L182)

This PR should not get merged in until #33 gets resolved (or it could be fixed in this PR).